Cargo.toml

```diff
@@ -24,8 +24,8 @@ garcon = { version = "0.2.3", features = ["async"] }
 hex = "0.4.3"
 hyper = { version = "0.14.13", features = ["full"] }
 hyper-tls = "0.5.0"
-ic-agent = "0.9"
-ic-utils = "0.7"
+ic-agent = { git = "https://github.com/paulyoung/agent-rs", branch = "paulyoung/http-request" }
+ic-utils = { git = "https://github.com/paulyoung/agent-rs", branch = "paulyoung/http-request" }
```

Comment:
These should be updated before merging. Ideally https://github.com/dfinity/agent-rs/pull/291 would be merged and then new versions tagged.

Reply:
Waiting on https://github.com/dfinity/agent-rs/pull/292 before using `10.0.2` here.